### PR TITLE
Create/button

### PR DESCRIPTION
**Pull request by heidi-ho07**
Thanks for your time <3

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  padding: 5px 5px 5px 20px;
+  margin-left: 15px;
+`
+
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+`
+
+function Button({ icon, onClick, ...props }) {
```
Comment (badazzdev) [inline src/components/Button.js]: You dont pass icon, onClick, ...props to this component either use it, so its useless. Please Delete them!

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  return (
+    <StyledContainer>
+      <Link to="/overview">
+        <StyledButton onClick={onClick} {...props}>
```
Comment (badazzdev) [inline src/components/Button.js]: Like above comment. delete onclick and ...props

File: src/components/Landingpage.js [hunk 3]
```diff
@@ -44,6 +46,7 @@ function Landingpage() {
           Ich bin super gespannt auf deine Erlebnisse. Klicke auf den Button und
           fange an mir von deinen Abenteuern zu erzählen.
         </StyledContent>
+        <Button to="/overview" />
```
Comment (badazzdev) [inline src/components/Landingpage.js]: This makes absolut no senese... please remove `to="/overview"`

File: src/components/Button.js (hunk 1 above)
Comment (heidi-ho07) [reply src/components/Button.js]: Yeah removed both, thought they were useful.

File: src/components/Landingpage.js (hunk 3 above)
Comment (heidi-ho07) [reply src/components/Landingpage.js]: Left that by testing...

**Pull request merged**